### ISSUE TITLE
Report on runtimes

### ISSUE THREAD

**Issue by jwodder** [dandi/dandisets-healthstatus#29]
@yarikoptic: You previously asked me to run the healthchecks on all Dandisets and record the time it took.  I can't find where that request was made, so I'm starting a new issue to record the results.

* A run of commit 8f71fb7 when the highest Dandiset ID was 401, processing 8 Dandisets at once, took 15653m 52.795s (10d 20h 53m 52s), finishing at 2023-01-14 09:38:06.

**Comment by yarikoptic** [comment]
Great, thanks - I think it was just in slack communication - we wanted to get some idea on duration to see how scalable etc it would be.  I guess we could add to `status.yaml` some metadata on when it started/finished for that particular run which produced that status.yaml.  It would not give us overall wall clock time if we run them in parallel but we could take min/max from all of those start/finish from a run through all to get it.

**Comment by yarikoptic** [comment]
We already have `last_run` time stamp.  It is not clear to me when it is exactly populated though -- before or after (only during dumping) the run. Please
- clarify when it is populated
- compliment with 
  - the alternative, e.g. `last_run_ended` if `last_run` depicts when it started, or `last_run_started` if it depicts when it ended
  - `last_run_duration` in seconds on how long it took

This would allow us to
- select dandisets which are "quick" to test
- figure out overall start/stop time across all dandisets if that would be desired.

**Comment by jwodder** [comment]
@yarikoptic `last_run` is set at the start of the run.

**Comment by jwodder** [comment]
Another data point:

* Running the healthchecks on just 000008, 000027, and 000029 using commit 8981bd6 (with the Dandisets parallelized and one asset per Dandiset processed at a time) took 1848m 18.387s.

**Comment by yarikoptic** [comment]
Thanks for the update. Yeah - slow, and most likely most of it on matnwb check -- seems to be quite slowish. But given that it was in parallel across dandisets, we could become bound by "slowest dandiset".  and complete in those few thousand minutes.
